CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### Version 2.6.2
```

Comment:
Version 2.7.0 

Unless I'm missing something, this is a minor release. From [Semantic Versioning 2.0.0](https://semver.org/#semantic-versioning-200)

> MINOR version when you add functionality in a backwards-compatible manner

Reply:
👍 still learning semantic lyfe. will update.